images/win/Windows2016-Azure.json

```diff
@@ -235,6 +235,14 @@
                 "{{ template_dir }}/scripts/Installers/Update-AndroidSDK.ps1"
             ]
         },
+        {
+            "type": "powershell",
+            "elevated_user": "SYSTEM",
```

Comment:
Just for curiosity, why we need `elevated_user` here?

Reply:
Some components can't be installed via winrm in that case we should use elevated_user parameter[ If specified, the PowerShell script will be run with elevated privileges using the given Windows user.]